Makefile

```diff
@@ -6,7 +6,8 @@ NETLIFY_FUNC      = $(NODE_BIN)/netlify-lambda
 # but this can be overridden when calling make, e.g.
 # CONTAINER_ENGINE=podman make container-image
 CONTAINER_ENGINE ?= docker
-CONTAINER_IMAGE   = kubernetes-hugo
+DOCKERFILE_VERSION = $(shell grep DOCKERFILE_VERSION Dockerfile | tail -n 1 | cut -d '=' -f 2 | tr -d " \"\n")
+CONTAINER_IMAGE   = kubernetes-hugo:v$(HUGO_VERSION)-$(DOCKERFILE_VERSION)
```

Comment:
TODO: `kubernetes-hugo` => something like `k8s.gcr.io/website/hugo` (not sure what the right name is just yet).

Reply:
Tagging the image based on these version details also gives us the basis of a future mechanism for someone to build the image locally and also rebuild it on major changes. Nice.